README.md

```diff
@@ -1,32 +1,43 @@
-# Create a JavaScript Action
+# Make state repo dispatches action
 
-[![GitHub Super-Linter](https://github.com/actions/javascript-action/actions/workflows/linter.yml/badge.svg)](https://github.com/super-linter/super-linter)
-![CI](https://github.com/actions/javascript-action/actions/workflows/ci.yml/badge.svg)
+# Inputs and configuration
 
-Use this template to bootstrap the creation of a JavaScript action. :rocket:
+This action dispatches changes made in a claims repo to one or more state repos. That is, when a change is made in a code repo, this action can be called to update the configurations of all related state repos by creating a PR with the relevant changes in each of them. In our case this usually means updating the `images.yaml` with a newly built image
 
-This template includes compilation support, tests, a validation workflow,
-publishing, and versioning guidance.
+## Inputs
 
-If you are new, there's also a simpler introduction in the
-[Hello world JavaScript action repository](https://github.com/actions/hello-world-javascript-action).
+The action takes the following inputs:
 
-## Create Your Own Action
+- `dispatches_file`: path to the `make_dispatches.yaml` in *your code repo*. Usually, and by default, `.github/make_dispatches.yaml`.
+- `token`: a Github token, with `repo` permissions over both the code and state repos. By default, the `github.token` of the action.
+- `image_type`: image type to dispatch. Can be any of `releases`, `snapshots` or `\*`, meaning *all* (default).
+- `flavors`: list of flavor names, as defined in the `dispatches_file`. Can be a single flavor or a list of comma separated flavors. By default, `default`.
+- `state_repo`: list of specific state repos to which to dispatch. Can be a single state repo name, a list of comma separated state repo names or `\*`, meaning *all* (default). These state repos must still be defined within the `dispatches_file`, i.e. this parameter works as a filter and won't make dispatches to "unknown" repositories.
+- `filter_by_env`: list of specific enviroments to which to dispatch. Can be a single environment name, a list of comma separated environment names or `\*`, meaning *all* (default). Since this parameter is a filter, the environment names must be one of those defined within the `dispatches_file`.
```

Comment:
```suggestion
- `filter_by_env`: list of specific environments to which to dispatch. Can be a single environment name, a list of comma separated environment names or `*`, meaning *all* (default). Since this parameter is a filter, the environment names must be one of those defined within the `dispatches_file`.
```